DataStructures/BitArray/TestBitArray.cs

```diff
@@ -0,0 +1,377 @@
+﻿/**
+ * Author: Christian Bender
+ * Class: TestBitArray
+ * 
+ * This class contains some tests for the class BitArray.
+ * 
+ * */
+
+
+using NUnit.Framework;
```

Comment:
The projects unit tests were refactored recently to use the XUnit framework. Could you please refactor your test suite to use XUnit instead of NUnit?